app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventPresenter.java

```diff
@@ -1,5 +1,7 @@
 package org.fossasia.openevent.app.core.event.create;
 
+import android.content.res.Resources;
```

Comment:
No `android.*` import is allowed in a presenter

Reply:
I'll need context to get the default locale. View can't handle logic. 
Do I need to inject context?